test/libsolidity/ASTImportTest.cpp

```diff
@@ -77,17 +77,14 @@ ASTImportTest::ASTImportTest(string const& _filename)
 	}
 
 	// save entire Json from input as string in m_expectations
-	// TODO but modify the metadata to not includes the hash of the solidity-source code
-	// TODO2: byteCodeSansMetadata might need to be modified to remove metadata of imported contracts too
 	if ( (*ast).isMember("contracts") )
 	{
 		for (auto& srcName: (*ast)["contracts"].getMemberNames())
 		{
-			// why is this not importing correctly? HELP namesspace issues?
-//			string bin = (*ast)["contracts"][srcName]["bin"].asString();
-//			string binSansMetadata = dev::test::bytecodeSansMetadata(bin);
-			// dummy
-			string binSansMetadata = "0x00";
+			// TODO somehow this still is an undefined reference, but bytecodeSansMetadata, being in the same file, is not :/
+//			string binSansMetadata = dev::test::bytecodeSansAnyMetadata((*ast)["contracts"][srcName]["bin"].asString());
```

Comment:
somehow this function can not be found when compiled (but the other one can):

![image](https://user-images.githubusercontent.com/9882716/62211579-33199f80-b39f-11e9-820b-dc7c4b4a9da7.png)